flink-runtime/src/test/java/org/apache/flink/runtime/state/filesystem/FileStateHandleTest.java

```diff
@@ -147,7 +144,7 @@ public void testDiscardStateWithDeletionFailureThroughException() throws Excepti
     }
 
     @Test
-    public void testDiscardStateWithDeletionFailureThroughReturnValue() throws Exception {
+    void testDiscardStateWithDeletionFailureThroughReturnValue() throws Exception {
```

Comment:
The public of `testDiscardStateWithDeletionFailureThroughException`can be removed.

And the public of `FsCheckpointStreamFactoryTest` class can be removed.